awscognito/create_new_user.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+AWS_USER_POOL_ID="us-east-1_lr10OX54W"
+AWS_PROFILE="virtual-acl2020"
```

Comment:
Because I have multiple AWS credentials in my environment, I have to switch the profile.